tests/test_views_collections.py

```diff
@@ -158,6 +158,10 @@ def test_can_be_created_after_deletion_with_if_none_match_star(self):
         self.app.put_json(self.collection_url, MINIMALIST_COLLECTION,
                           headers=headers, status=201)
 
+    def test_records_permissions_are_removed_after_collection_deleted(self):
+        self.assertDictEqual(self.permission.get_object_permissions(
```

Comment:
To test this, you would need to actually create someting and delete it from the test function itself.

Reply:
@Natim I thought the creation is covered in _setUp_ method:

```python
    def setUp(self):
        super().setUp()
        bucket = {**MINIMALIST_BUCKET,
                  'permissions': {'collection:create': ['system.Everyone'],
                                  'read': ['system.Everyone']}}
        self.app.put_json('/buckets/beers', bucket,
                          headers=self.headers)
        self.app.put_json(self.collection_url, MINIMALIST_COLLECTION,
                          headers=self.headers)
        resp = self.app.post_json(self.collection_url + '/records',
                                  MINIMALIST_RECORD,
                                  headers=self.headers)
        self.previous_ts = resp.headers['ETag']
        record_id = resp.json['data']['id']
        self.record_url = self.collection_url + '/records/{}'.format(record_id)
        self.app.delete(self.collection_url, headers=self.headers)
```
Do I still need to implement "create/delete" explicitly?

Reply:
I would say that it is good enough for me then. Can you also make sure that the collection permissions is empty?

Reply:
Sure, will update PR later today